lib/iris/quickplot.py

```diff
@@ -303,6 +336,11 @@ def scatter(x, y, *args, **kwargs):
     See :func:`iris.plot.scatter` for details of valid arguments and
     keyword arguments.
 
+    Notes
+    ------
+    This function does not maintain laziness when called; it realises data.
+    See more at :doc:`/userguide/real_and_lazy_data`.
+
     """
```

Comment:
@ESadek-MO There are no statements about laziness for the following `iris.quickplot` public API functions:

- [ ] `fill_between`
- [ ] `hist`

Is this because they do maintain laziness?